rest-server/bin/harness-engine-import

```diff
@@ -7,35 +7,32 @@
 
 while [ -n "$1" ]; do
     case "$1" in
-        *) RESOURCE_ID=$1
-            FILENAME=$2
-            #echo "Filename: ${FILENAME}"
+        *) resource_id=$1
+            input_path=$2
+            echo "Engine id: ${resource_id} filename: ${input_path}"
             ;;
```

Comment:
moving to the convention of small case bash vars when they are never kept in the env.